Kinetic_data_structures/include/CGAL/Kinetic/Default_simulator.h

```diff
@@ -269,7 +269,7 @@ class Default_simulator: public Ref_counted<Default_simulator<Polynomial_kernel_
     return audit_time_;
 #else
     CGAL_precondition(0);
-    static NT z(0);
+    static const NT z(0);
```

Comment:
Just a remark, that I keep repeating so that we remember it well: if the `NT` is not a POD, in particular if that is a `Lazy_exact_nt`, the initialization of that `static const` is not thread-safe before C++11.

I do not say that we modify again the code, but I want to point out, again, that the thread-safety is not solved unless C++11 is used.